akka-remote/src/main/scala/akka/remote/RemoteActorRefProvider.scala

```diff
@@ -59,10 +60,15 @@ private[akka] object RemoteActorRefProvider {
     }
 
     when(WaitTransportShutdown) {
-      case Event((), _) ⇒
+      case Event(Done, _) ⇒
         log.info("Remoting shut down.")
         systemGuardian ! TerminationHookDone
         stop()
+
+      case Event(Status.Failure(ex), _) ⇒
+        log.error(ex, "Remoting shut down with error")
```

Comment:
when will this happen? is it ask timeout?
I'm thinking if the log level should be error, warning or info.